reya-rust-sdk/src/http_provider.rs

```diff
@@ -282,4 +363,28 @@ impl HttpProvider {
 
         eyre::Ok(vec![])
     }
+
+    ///
+    /// get the current pool price by market id and returns the instantaneous pool price
+    ///
+    pub async fn get_pool_price(&self, market_id: u128) -> eyre::Result<U256> {
+        // create http provider
+        let provider = ProviderBuilder::new()
+            .with_recommended_fillers()
+            .on_http(self.sdk_config.rpc_sdk_url.clone());
+
+
+        let core_proxy = CoreProxy::new(
```

Comment:
should be

let passive_perp_proxy = PassivePerpProxy:new...

we should use the passive perp contract address